lib/__tests__/__snapshots__/cli.test.js.snap

```diff
@@ -9,103 +9,105 @@ exports[`CLI --help 1`] = `
 
-    --custom-syntax
+    --custom-syntax <name_or_path>
 
-      Module name or path to a JS file exporting a PostCSS-compatible syntax.
+      A module name or path to a JS file exporting a PostCSS-compatible syntax.
 
     --stdin
 
       Accept stdin input even if it is empty.
 
-    --stdin-filename
+    --stdin-filename <name>
 
       A filename to assign stdin input.
 
     --ignore-disables, --id
 
-      Ignore stylelint-disable comments.
+      Ignore "stylelint-disable" comments.
 
     --disable-default-ignores, --di
 
-      Allow linting of node_modules.
+      Allow linting of "node_modules".
 
-    --cache                       [default: false]
+    --[no-]cache
 
       Store the info about processed files in order to only operate on the
-      changed ones the next time you run stylelint. By default, the cache
-      is stored in "./.stylelintcache". To adjust this, use --cache-location.
+      changed ones the next time you run Stylelint. By default, the cache is
+      stored in "./.stylelintcache". To adjust this, use "--cache-location".
+      Cache is disabled by default.
 
-    --cache-location              [default: '.stylelintcache']
+    --cache-location <path>
 
-      Path to a file or directory to be used for the cache location.
-      Default is "./.stylelintcache". If a directory is specified, a cache
-      file will be created inside the specified folder, with a name derived
-      from a hash of the current working directory.
+      A path to a file or directory to be used for the cache location. If a
+      directory is specified, a cache file will be created inside the specified
+      folder, with a name derived from a hash of the current working directory.
 
       If the directory for the cache does not exist, make sure you add a trailing "/"
-      on *nix systems or "\\" on Windows. Otherwise the path will be assumed to be a file.
+      on *nix systems or "\\" on Windows. Otherwise the path will be assumed to
+      be a file.
 
-    --cache-strategy              [default: "metadata"]
+    --cache-strategy <metadata|content>
 
-      Strategy for the cache to use for detecting changed files. Can be either
-      "metadata" or "content".
+      A strategy for the cache to use for detecting changed files. Defaults to "metadata".
 
-      The "content" strategy can be useful in cases where the modification time of
-      your files changes even if their contents have not. For example, this can happen
-      during git operations like "git clone" because git does not track file modification
-      time.
+      The "content" strategy can be useful in cases where the modification time
+      of your files changes even if their contents have not. For example, this can
+      happen during git operations like "git clone" because git does not track file
+      modification time.
 
-    --formatter, -f               [default: "string"]
+    --formatter, -f <compact|github|json|string|tap|unix|verbose>
```

Comment:
Agh! Retrying to post this comment again.

---

A minor question: it is possible that users confuse this argument notation (`<>`, where each item is a literal) opposed to other cases where `<>` describes a type.

For example, the `<...|json|...>` here describes the literal string `"json"`; but, for `--go <json>`, that refers to a set of values that is valid JSON.

(this would also apply to `--cache-strategy`)

The `git-log` docs have a few possible options:

- using braces (`{}`), which feels intuitive like set notation; ref [`--diff-algorithm`](https://www.git-scm.com/docs/git-log#Documentation/git-log.txt---diff-algorithmpatienceminimalhistogrammyers)
- using parens (`()`), which feel intuitive like regex/tuples; ref [`--diff-merges`](https://www.git-scm.com/docs/git-log#Documentation/git-log.txt---diff-mergesoffnoneonfirst-parent1separatemcombinedcdense-combinedccremerger)
- defining a new `<format>` type, which is what they do for their [`--format`](https://www.git-scm.com/docs/git-log#Documentation/git-log.txt---formatltformatgt)

What do you think? I also think leaving it as-is is generally reasonable.

Reply:
Thanks, good point! Listing values sounds more understandable like [`--diff-algorithm`](https://www.git-scm.com/docs/git-log#Documentation/git-log.txt---diff-algorithmpatienceminimalhistogrammyers). For example:

```
--formatter, -f <formatter>

  Choose an output formatter. The variants are as follows:
  
    string - human-readable strings (default)
    compact - similar to ESLint's compact formatter 
    github - workflow commands for GitHub Actions
    json - JSON format
    tap - TAP format
    unix - C compiler-like format
    verbose - a file list and a tally, extending the "string" formatter
```

What do you think?

Ref:

https://github.com/stylelint/stylelint/blob/231dac450f345f84966983db5bc6cce3b3e5989a/docs/user-guide/options.md?plain=1#L85-L91

Reply:
I think that's a great idea! LGTM.

Reply:
Thanks, I changed it via d27ea14d.